File: library/src/main/java/com/mikepenz/materialdrawer/model/interfaces/Typefaceable.java
package com.mikepenz.materialdrawer.model.interfaces;

import android.graphics.Typeface;

/**
 * Created by mikepenz on 03.02.15.
 */
public interface Typefaceable<T> {
    T withTypeface(Typeface typeface);

    Typeface getTypeface();
}

File: library/src/main/java/com/mikepenz/materialdrawer/model/interfaces/Typefaceable.kt
package com.mikepenz.materialdrawer.model.interfaces

import android.graphics.Typeface

/**
 * Created by mikepenz on 03.02.15.
 */
interface Typefaceable<T> {

    val typeface: Typeface?

    fun withTypeface(typeface: Typeface?): T
}
